### ISSUE TITLE
make gives `gometalinter.v1: error: failed to install the following linters:`

### ISSUE THREAD

**Issue by TomSweeneyRedHat** [containers/buildah#1704]
**Description**
When doing `make` for the Buildah code from GitHub, this error is received: 
```
gometalinter.v1: error: failed to install the following linters:
```

**Steps to reproduce the issue:**
1. 'make`

**Describe the results you received:**
See description.

**Describe the results you expected:**
Clean make.

**Comment by TomSweeneyRedHat** [comment]
Probable fix to Makefile in #1703

**Comment by TomSweeneyRedHat** [comment]
This has been cleaned up, closing.
